google-cloud-bigquery-data_transfer/lib/google/cloud/bigquery/data_transfer/v1/data_transfer_service_client.rb

```diff
@@ -0,0 +1,860 @@
+                defaults["get_data_source"]
+              )
+              @list_data_sources = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:list_data_sources),
+                defaults["list_data_sources"]
+              )
+              @create_transfer_config = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:create_transfer_config),
+                defaults["create_transfer_config"]
+              )
+              @update_transfer_config = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:update_transfer_config),
+                defaults["update_transfer_config"]
+              )
+              @delete_transfer_config = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:delete_transfer_config),
+                defaults["delete_transfer_config"]
+              )
+              @get_transfer_config = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:get_transfer_config),
+                defaults["get_transfer_config"]
+              )
+              @list_transfer_configs = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:list_transfer_configs),
+                defaults["list_transfer_configs"]
+              )
+              @schedule_transfer_runs = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:schedule_transfer_runs),
+                defaults["schedule_transfer_runs"]
+              )
+              @get_transfer_run = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:get_transfer_run),
+                defaults["get_transfer_run"]
+              )
+              @delete_transfer_run = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:delete_transfer_run),
+                defaults["delete_transfer_run"]
+              )
+              @list_transfer_runs = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:list_transfer_runs),
+                defaults["list_transfer_runs"]
+              )
+              @list_transfer_logs = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:list_transfer_logs),
+                defaults["list_transfer_logs"]
+              )
+              @check_valid_creds = Google::Gax.create_api_call(
+                @data_transfer_service_stub.method(:check_valid_creds),
+                defaults["check_valid_creds"]
+              )
+            end
+
+            # Service calls
+
+            # Retrieves a supported data source and returns its settings,
+            # which can be used for UI rendering.
+            #
+            # @param name [String]
+            #   The field will contain name of the resource requested, for example:
+            #   +projects/{project_id}/dataSources/{data_source_id}+
```

Comment:
It appears that these comments that are copied from proto are interpreted as links in yarddoc. @blowmage what would you expect this line to be transformed to in order for this to be valid yarddoc?

Reply:
Hm. I don't think theres an easy way to automate that kind of alteration to that. maybe is there some sort of way to simply alter the bracketed portion?

Reply:
Ah saw that this is just emitting a warning and that the CI errors were due to the lack of docs/toc.json. I added that file in my latest commit.